### PR TITLE
PWN-7913 - SVL. Links history text fix

### PR DESCRIPTION
**Pull request by gslevinkov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-7913

### CODE REVIEW

File: ui-kit/src/main/kotlin/org/p2p/uikit/components/finance_block/FinanceBlockDelegates.kt [hunk 1]
```diff
@@ -4,18 +4,28 @@ import com.hannesdorfmann.adapterdelegates4.AdapterDelegate
...
+private typealias InflateListener = (financeBlock: UiKitFinanceBlockView) -> Unit
+private typealias OnBindListener = (view: UiKitFinanceBlockView, item: FinanceBlockCellModel) -> Unit
+private typealias OnItemClickedListener = (item: FinanceBlockCellModel) -> Unit
 
 fun financeBlockCellDelegate(
-    inflateListener: ((financeBlock: UiKitFinanceBlockView) -> Unit)? = null,
-    onBindListener: ((view: UiKitFinanceBlockView, item: FinanceBlockCellModel) -> Unit)? = null,
+    inflateListener: InflateListener? = null,
+    onBindListener: OnBindListener? = null,
+    onItemClicked: OnItemClickedListener? = null
```
Comment (gslevinkov) [inline ui-kit/src/main/kotlin/org/p2p/uikit/components/finance_block/FinanceBlockDelegates.kt:16]: @NikolaiEmelianov fyi

File: ui-kit/src/main/kotlin/org/p2p/uikit/components/right_side/UiKitRightSideView.kt [hunk 2]
```diff
@@ -82,6 +82,9 @@ class UiKitRightSideView @JvmOverloads constructor(
     private fun WidgetRightSideDoubleTextBinding.bind(model: RightSideCellModel.TwoLineText) {
         this.textViewFirst.bindOrGone(model.firstLineText)
         this.textViewSecond.bindOrGone(model.secondLineText)
+        if (model.secondLineText != null) {
```
Comment (gslevinkov) [inline ui-kit/src/main/kotlin/org/p2p/uikit/components/right_side/UiKitRightSideView.kt:85]: @NikolaiEmelianov fyi

File: ui-kit/src/main/res/layout/widget_finance_block.xml [hunk 3]
```diff
@@ -19,22 +19,25 @@
         app:layout_constrainedHeight="true"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toStartOf="@+id/rightSideView"
-        app:layout_constraintHorizontal_bias="0.5"
-        app:layout_goneMarginEnd="0dp"
+        app:layout_constraintHorizontal_bias="0.9"
         app:layout_constraintHorizontal_chainStyle="spread_inside"
+        app:layout_constraintHorizontal_weight="0.7"
```
Comment (gslevinkov) [inline ui-kit/src/main/res/layout/widget_finance_block.xml:24]: @NikolaiEmelianov fyi

File: ui-kit/src/main/kotlin/org/p2p/uikit/components/finance_block/FinanceBlockDelegates.kt [hunk 4]
```diff
@@ -4,18 +4,28 @@ import com.hannesdorfmann.adapterdelegates4.AdapterDelegate
...
-        viewBinding = { inflater, parent -> ItemFinanceBlockBinding.inflate(inflater, parent, false) },
+        viewBinding = { _, parent -> parent.inflateViewBinding(attachToRoot = false) },
         on = { item, _, _ -> item is FinanceBlockCellModel && item.styleType == FinanceBlockStyle.FINANCE_BLOCK }
     ) {
 
         binding.root.bindViewStyle(FinanceBlockStyle.FINANCE_BLOCK)
         inflateListener?.invoke(binding.root)
+        if (onItemClicked != null) {
+            binding.root.setOnClickListener { onItemClicked.invoke(item) }
+        }
```
Comment (1sakov) [inline ui-kit/src/main/kotlin/org/p2p/uikit/components/finance_block/FinanceBlockDelegates.kt:27]: `binding.root.setOnClickListener { onItemClicked?.invoke(item) }`

**Review by 1sakov — APPROVED** [review]
minor comments

**Review by kabramovp2p — APPROVED** [review]

**Pull request merged**